src/printer-postcss.js

```diff
@@ -34,35 +34,51 @@ function genericPrint(path, options, print) {
       if (n.raws.content) {
         return n.raws.content;
       }
+      const isIndentStyle = options.indentStyles
```

Comment:
`isIndentStyle` sounds like a boolean to me, but it seems to be a function.

Reply:
Yep it's a function. Would `indentStyles` or perhaps `isIndentStyleFn` be a better name, even when `options.indentStyles` is `false`?

I suppose I was thinking about the later use of the function when it's wrapping `isIndentStyle(concat(...))` to show that it might be doing nothing (just using a pass-through) when the option is `false`. Originally I didn't have a pass-through and it was a conventional `const formatted = concat(/* the existing concat */); return options.indentStyles ? getIndentStyle(path, options, print)(formatted) : formatted;` but for the `css-comment` area this would mean more duplicate code.

Reply:
`maybeIndentStyle` ?

Reply:
Changed!